docs/docs.md

```diff
@@ -8,6 +8,23 @@ and sharing them with others. You can watch [this video](https://www.youtube.com
 
 Run games by hitting the `Run` button or pressing `Shift+Enter`.
 
+### Infinite loop detection
```

Comment:
Suggest cutting this.  Users will experience it intuitively and this probably shouldn't be above the "Getting Help" section.